editoast/src/views/rolling_stocks.rs

```diff
@@ -621,4 +717,148 @@ pub mod tests {
         //Delete rolling_stock
         call_service(&app, rolling_stock_delete_request(rolling_stock_id)).await;
     }
+
+    #[rstest]
+    async fn check_usage_rolling_stock_not_found() {
+        let app = create_test_service().await;
+        let rolling_stock_id = 314159;
+        let response = call_service(
+            &app,
+            TestRequest::get()
+                .uri(format!("/rolling_stock/{}/check_usage", rolling_stock_id).as_str())
+                .to_request(),
+        )
+        .await;
+        assert_eq!(response.status(), StatusCode::NOT_FOUND);
+        assert_editoast_error_type!(response, RollingStockError::NotFound { rolling_stock_id });
+    }
+
+    #[rstest]
+    async fn check_usage_no_train_schedule_for_this_rolling_stock(
+        #[future] fast_rolling_stock: TestFixture<RollingStockModel>,
+    ) {
+        let rolling_stock = fast_rolling_stock.await;
+        let app = create_test_service().await;
+        let rolling_stock_id = rolling_stock.id();
+        let response = call_service(
+            &app,
+            TestRequest::get()
+                .uri(format!("/rolling_stock/{}/check_usage", rolling_stock_id).as_str())
+                .to_request(),
+        )
+        .await;
+        let response_body: UsageResponse = assert_status_and_read!(response, StatusCode::OK);
+        let expected_body = UsageResponse { usage: Vec::new() };
+        assert_eq!(response_body, expected_body);
+    }
+
+    /// `check_usage_one_train_schedule` checks the operation of the `/rolling_stock/{rolling_stock_id}/check_usage` endpoint.
+    /// There exist one `TrainSchedule` that uses the give `rolling_stock_id`.
+    /// The endpoint should find that `TrainSchedule` from this `rolling_stock_id`
+    /// as well the project/study/scenario where it’s defined
+    /// and should return their ids/names
```

Comment:
This docstring is surprisingly long and verbose, it feels like a relic of github copilot ?

Reply:
It’s a already simplified description I asked chatGPT to give me about this test.